packages/shared/lib/models/Sync.ts

```diff
@@ -125,6 +126,7 @@ export interface SlimSync {
     sync_id?: string | null;
     providerConfigKey?: string;
     connections?: number;
+    enabled?: boolean;
```

Comment:
The problem with mixing typing here is that it's optional at creation but not when you query

Reply:
This is for a `SlimSync` where it is optional and it isn't queried for but rather is a constructed object where `enabled` isn't always needed.

Reply:
It is used in `getSyncConfigsByProviderConfigKey` which now output enabled by default